python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml

```
@@ -15,6 +15,9 @@ keys:
       The sFlow process will only be configured if any interface is enabled for sFlow.
       For default enabling of sFlow for various interface types across the fabric see `fabric_sflow`.
     keys:
+      polling_interval:
+        type: int
+        description: Interval for sending counter data to the sFlow collector
```

For int we always add convert_types from str
```suggestion
        convert_types:
          - str
        description: Interval for sending counter data to the sFlow collector.
```

---

Updated